src/components/Reference.js

```diff
@@ -4,7 +4,7 @@ import Link from './Link';
 import { getNestedValue } from '../utils/get-nested-value';
 
 const Reference = ({ nodeData }) => (
-    <Link className="reference external" to={nodeData.refuri}>
+    <Link className="reference" to={nodeData.refuri} target="_blank">
```

Comment:
Yeah eventually we can swap these links to ours. Cool test for internal here though:
`const external = /^http(s)?:\/\//.test(to);`